src/Core/Cli.re

```diff
@@ -5,6 +5,9 @@
  */
 open Rench;
 
+module CoreLog = Log;
+module Log = (val Log.withNamespace("Oni2.Core.Cli"));
```

Comment:
I've added the project as an extra namespaces as well, since ti disambiguates a few things and even allows some namespaces to be shorter if some of the redundancy is removed.